src/main/java/io/cdap/e2e/utils/CdfHelper.java

```diff
@@ -100,12 +101,12 @@ default void succeedPipelineVerification() {
 
   default int recordIn() {
     String incount = CdfPipelineRunLocators.recordsInCount.getText();
-    return Integer.parseInt(incount);
+    return Integer.valueOf(incount.replaceAll(",", StringUtils.EMPTY).toString());
   }
 
   default int recordOut() {
     String outcount = CdfPipelineRunLocators.recordsOutCount.getText();
-    return Integer.parseInt(outcount);
+    return Integer.valueOf(outcount.replaceAll(",", StringUtils.EMPTY).toString());
```

Comment:
Why are you switching from `parseInt` to `valueOf`? Also I think `.toString()` is not needed

Reply:
Changed to parseInt and removed redundant .toString()